packages/@lwc/engine-core/src/framework/wiring.ts

```diff
@@ -315,8 +359,7 @@ export function installWireAdapters(vm: VM) {
         def: { wire },
     } = vm;
 
-    const wiredConnecting = (context.wiredConnecting = []);
-    const wiredDisconnecting = (context.wiredDisconnecting = []);
+    const wiredConnectors: WireConnector[] = (context.wiredConnectors = []);
```

Comment:
```suggestion
    const wiredConnectors = context.wiredConnectors;
```

Reply:
From what I understand, a new array has to be allocated here. Until this point, all the VM has shared the same reference to an empty array. A comment should be added here for sure.